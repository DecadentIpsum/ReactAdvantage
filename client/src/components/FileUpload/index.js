import React, { Component } from 'react';
import { FileUpload as PrimeFileUpload } from 'primereact/components/fileupload/FileUpload';

export default class FileUpload extends Component {
    render() {
        return (
            <PrimeFileUpload {...this.props} />
        );
    }
}
