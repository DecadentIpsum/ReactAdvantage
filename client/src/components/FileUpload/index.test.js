import React from 'react';
import Enzyme, { shallow } from 'enzyme';

import FileUpload from './index';

it('renders correctly', () => {
    const component = shallow(<FileUpload />);

    expect(component).toMatchSnapshot();
});
