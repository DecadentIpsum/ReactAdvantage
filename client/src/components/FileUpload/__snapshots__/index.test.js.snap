// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FileUpload />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "accept": null,
      "auto": false,
      "cancelLabel": "Cancel",
      "chooseLabel": "Choose",
      "className": null,
      "disabled": false,
      "id": null,
      "invalidFileSizeMessageDetail": "maximum upload size is {0}.",
      "invalidFileSizeMessageSummary": "{0}: Invalid file size, ",
      "maxFileSize": null,
      "mode": "advanced",
      "multiple": false,
      "name": null,
      "onBeforeSend": null,
      "onBeforeUpload": null,
      "onClear": null,
      "onError": null,
      "onProgress": null,
      "onSelect": null,
      "onUpload": null,
      "previewWidth": 50,
      "style": null,
      "uploadLabel": "Upload",
      "url": null,
      "widthCredentials": false,
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "accept": null,
        "auto": false,
        "cancelLabel": "Cancel",
        "chooseLabel": "Choose",
        "className": null,
        "disabled": false,
        "id": null,
        "invalidFileSizeMessageDetail": "maximum upload size is {0}.",
        "invalidFileSizeMessageSummary": "{0}: Invalid file size, ",
        "maxFileSize": null,
        "mode": "advanced",
        "multiple": false,
        "name": null,
        "onBeforeSend": null,
        "onBeforeUpload": null,
        "onClear": null,
        "onError": null,
        "onProgress": null,
        "onSelect": null,
        "onUpload": null,
        "previewWidth": 50,
        "style": null,
        "uploadLabel": "Upload",
        "url": null,
        "widthCredentials": false,
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
